src/sentry/auth/system.py

```diff
@@ -59,7 +59,7 @@ def __hash__(self):
         return 1  # singleton
 
     def is_expired(self):
-        False
+        return False
```

Comment:
lol

Reply:
~technically it still used to work due to implicit `return None` being falsey, v lucky